docs/commands/context.md

```diff
@@ -63,6 +63,31 @@ tracking the evolution of arbitrary locations in memory. Tracked locations can
 be removed one by one using `memory unwatch`, or altogether with `memory reset`.
 
 
+To have the stack the largest stack addresses on top (i.e., grow the stack downward), enable the following setting
+```
+gef➤ gef config context.grow_stack_down True
+```
+
+To always display the saved instruction pointer in the stack view, enable the following:
+```
+gef➤ gef config context.show_saved_ip True
+```
+
+This looks like the following when enabled, the addresses between the frame pointer and esp squished into an ellipsis:
```

Comment:
`esp` is arch specific: Just say "between the frame pointer and the stack pointer"